### PR TITLE
SPM-769  feat(git commits): use commitizer to enable conventionam commit naming

### PR DESCRIPTION
**Pull request by mkholjuraev**
This is a test PR from using commitizer for the ticket https://issues.redhat.com/browse/SPM-769

[cz-cli](https://github.com/commitizen) has several steps to finish the commit e.g type, scope, footer .... I thought some of them are unnecessaryy, thus I used [cz-customizable](https://github.com/leonardoanalista/cz-customizable) with custom config which can be improved according to needs later.

### CODE REVIEW

File: .cz-config.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                'chore:    Changes to the build process or auxiliary tools\n            and libraries such as documentation generation',
+        },
+        { value: 'docs', name: 'docs:     Documentation only changes' },
+        {
+            value: 'style',
+            name:
+                'style:    Changes that do not affect the meaning of the code\n            (white-space, formatting, missing semi-colons, etc)',
+        },
+        {
+            value: 'refactor',
```
Comment (jiridostal) [inline .cz-config.js:18]: Maybe we should add this rule to semantic-release, so it triggers a "patch"(fix) build.

File: .cz-config.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            name: 'refactor: A code change that neither fixes a bug nor adds a feature',
+        },
+        {
+            value: 'perf',
```
Comment (jiridostal) [inline .cz-config.js:22]: Same as above, let's treat this as a fix in semantic release

File: .cz-config.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    scopes: [],
+
+    allowTicketNumber: true,
+    isTicketNumberRequired: true,
+    ticketNumberPrefix: 'SPM-',
+    ticketNumberRegExp: '',
+
+    // override the messages, defaults are as follows
+    messages: {
```
Comment (jiridostal) [inline .cz-config.js:38]: The "Ticket number" message should say that "SPM" is added automatically

**Review by jiridostal — CHANGES_REQUESTED** [review]
Looks and works really great, just some minor improvements!

File: .cz-config.js (hunk 1 above)
Comment (mkholjuraev) [reply .cz-config.js:18]: I see there is 'refactor' in semantic release config https://github.com/RedHatInsights/patchman-ui/blob/master/package.json#L137. Am I looking at the right place? I am not very experienced with semantic release config :)

**Pull request merged**